src/components/User/User.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const User = ({ user }) => (
+  <>
+    <th>{user.name}</th>
+  </>
```

Comment:
Якщо повертаєш лише один елемент строінки то думаю цей пустий фрагмент не варто марно писати ..
```suggestion
  <th>{user.name}</th>
```